i6_models/config.py

```diff
@@ -29,7 +29,23 @@ class ModelConfiguration:
     def _validate_types(self) -> None:
         for field in fields(type(self)):
             try:
-                typeguard.check_type(getattr(self, field.name), field.type)
+                if isinstance(field.type, str):
```

Comment:
Hmm, why is this marked new? This is in main right?

Reply:
Better rebase it.